stock_valued_picking_report/report/stock_picking_valued_report.xml

```diff
@@ -13,7 +13,9 @@
                 <th class="text-right"><strong>Taxes</strong></th>
             </t>
         </xpath>
-        <xpath expr="//span[@t-field='pack_operation.product_uom_id']/.." position="after">
+        <xpath
+                expr="//span[@t-field='pack_operation.qty_done_uom_ordered']/."
```

Comment:
Dot missed

Reply:
We are returned to first commit 45 day ago:
https://github.com/OCA/stock-logistics-reporting/pull/38/commits/41a15ebc2d3c101b2537c5afacc1d0f9cca9d825

Read my last month comment https://github.com/OCA/stock-logistics-reporting/pull/38/commits/41a15ebc2d3c101b2537c5afacc1d0f9cca9d825#r145897719